datahub-web-react/src/app/entity/shared/types.ts

```diff
@@ -21,13 +21,18 @@ import {
 } from '../../../types.generated';
 import { FetchedEntity } from '../../lineage/types';
 
+export type GenericEntityTabProperties = {
+    editMode: boolean;
+};
```

Comment:
thinking about it again, you were right to have typed this as `any`. Let's go back to that.

Reply:
done